ovs/lib/graphite.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (C) 2018 iNuron NV
+#
+# This file is part of Open vStorage Open Source Edition (OSE),
+# as available from
+#
+#      http://www.openvstorage.org and
+#      http://www.openvstorage.com.
+#
+# This file is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License v3 (GNU AGPLv3)
+# as published by the Free Software Foundation, in version 3 as it comes
+# in the LICENSE.txt file of the Open vStorage OSE distribution.
+#
+# Open vStorage is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY of any kind.
+
+import time
+from ovs.extensions.generic.graphiteclient import GraphiteClient
+from ovs.extensions.generic.configuration import Configuration
+
+
+class GraphiteController(GraphiteClient):
```

Comment:
Not seeing the added value of this class
It inherits from GraphiteClient but it never initializes itself. It also build a new client each time?
Just remove this client and add the methods within the scrubber itself...

Reply:
The controller should bundle all calls, where the client does the connection work, and doesnt care what exactly he sends towards graphite. 
The graphiteclient beind rebuild everytime is bad though, indeed

Reply:
The controller should not inherit from the client but should be instantiated so it can make the graphite client for further use.
```
class GraphiteController(object):
    def __init__(self):
         self._graphite_client = GraphiteClient()
   // Scrubber stats
   def send...
```